### PR TITLE
Update storybook for button

### PR DESCRIPTION
**Pull request by SOLBI1028**
## Description

Add more samples for button component with source code.
And edit props table by adding necessary and default columns.
This image is an example.
image

## Related Issues

#57 

## Tests

I don't know well about test code. So please give me advice how can I test this PR. 🥲

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by luke-hanwook** [comment]
I am considering to enhance storybook documentation(eg. remove legacy props, add more usage example and section sorting etc...) For first, i think we need to get a new template format. what do you think about? why don't we reference some site?

**Comment by 2unbini** [comment]
I'm the one who's working with storybook documentation, and I agree with hanwook's opinion.

While adding some example codes of the components and making the storybook more readable(like matching the titles), I felt the need of the fixed template in storybook.
In my case, 'Accordion' has 'Installation' part, but other components don't. And also, the title 'Usage' is used for the specific example code in 'Accordion' page, but is used for the simple import-and-use code in other pages.

So I think setting the standard of the storybook is worth discussing.

**Comment by SOLBI1028** [comment]
> I am considering to enhance storybook documentation(eg. remove legacy props, add more usage example and section sorting etc...) For first, i think we need to get a new template format. what do you think about? why don't we reference some site?

I think it is good idea! I referred to bootstrap-vue site to add samples.
Do you have some good references??

**Comment by luke-hanwook** [comment]
We need to meet for sync :)
I will talk you later by slack!

File: docs/src/components/Button/Button.stories.mdx [hunk 1]
```diff
@@ -18,54 +18,227 @@ import {Button} from 'dooboo-ui';
...
+### `onPress` event
+When pressing button, some event can generate by using `onPress` prop.
+```tsx
+const [value, setValue] = useState(1)
+  const onIncrease = () => {
+    setValue(value + 1);
+  }
+  const onDecrease = () => {
+    setValue(value - 1);
+  }
```
Comment (yujong-lee) [inline docs/src/components/Button/Button.stories.mdx:135]: I recommend following [this naming convention](https://jaketrent.com/post/naming-event-handlers-react).

So you can name it `handlePressIncrease`.

Additionally, I personally like to write like this.

```jsx

const handlePress = (offset) => {setValue(value + offset)}

  <Button 
     text='Increase'
     style={{padding: 8}}
     onPress={() => handlePress(1)}
   />
   <Button 
     text='Decrease'
     style={{padding: 8}}
     onPress={() => handlePress(-1)}
   />
```

Why am I doing this? Less event handler is better? 
Maybe. But my point is on `offSet` naming. It makes code self-explainary.

If you want to show intention stronger, you can do something like this. (Although I think It is not needed here.)

```jsx
const handlePress = (offset) => {
  setValue((previous) => previous + offset)
}
```

**Review by yujong-lee — CHANGES_REQUESTED** [review]
Great work!

Like others said, `dooboo-ui` need more guidelines on many things.(code style, test style, rules, document style ...)

I think It might be very helpful for you to set up some guidelines on documentation. 

One thing I want to point out for further enhancement on document is, we always need to think about `Why`. Why ` bootstrap-vue` document is written like A? Why not B?  Why we need to show both form of `dark` mode and `light` mode for every example?

By thinking of Why, I think we can make lot better document than others.

**Review by hyochan — APPROVED** [review]
Looks great to me! Much better document and well organized!

However, I've made some fixes on lint that did not apply on `docs` correctly.
I might help you setup `eslint` and `prettier` config later.

I'll merge this `PR` since new linting is necessary to be applied ASAP. Thus, looks great enough.

Great work on this 💯

**Pull request merged**